### ISSUE TITLE
[Typst Universe] Add thumbnails to packages

### ISSUE THREAD

**Issue by rwmpelstilzchen** [typst/packages#1210]
### Description

Templates on the Typst Universe have thumbnails, which greatly help with distinguishing between them and getting a notion of what they do and how they look at a glance. I think it would be helpful for packages to have thumbnails too; currently they all have the same crate (`<div role="img" class="package-icon"></div>`) as their thumbnail.

Ideas for guidelines:

* Thumbnails should be clear even in small sizes.
* They should depict a logo or a representative cropped shot that demonstrates concisely what the package does, maximising the relevant information to clutter ratio.
* Care should be taken not to use shocking colours and patterns: draw attention to your package by providing good functionality and documentation, not by having a flashy thumbnail.

(Issue copied [from the Forum](https://forum.typst.app/t/548))

**Comment by reknih** [comment]
I'll just post what I have said in our internal chat. I think that your guidelines are good, but designs are particularly hard to get right.

![Chat history, post by reknih. 1 of 3: Why I did not do it at the time is to prevent being flooded with shitty design.
2 of 3: Although it's something I'm open to once the ecosystem matures.
3 of 3: And these logos need to support dark & light mode because I'm using the box both in black and white](https://github.com/user-attachments/assets/7444139b-ac1c-4ee0-b955-c5db1b46ab03)

**Comment by rwmpelstilzchen** [comment]
Yeah, I see your point (this is why I formulated the above guidelines, to counter shitty designs…). All submissions to the Typst Universe are reviewed before they are accepted or rejected. Thumbnails for packages (or templates for this matter) are not different to any other aspect of a submission, be it the code itself, the title, etc.

**Comment by Ants-Aare** [comment]
maybe make having a well designed icon a condition for packages who want to at some point leave the preview status.
